<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seconduser</class>
 <widget class="QWidget" name="seconduser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>334</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>NYSDAPP1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix1/LOGO NYSD.jpg</normaloff>:/new/prefix1/LOGO NYSD.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_18">
   <item>
    <widget class="QStatusBar" name="statusbar"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Algerian&quot;;</string>
     </property>
     <property name="text">
      <string>NOUMBISSI YAMDJEUSON STANLEY  DEREK                                                                                            @NYSD DEREK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QMenuBar" name="menubar"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="loginPage">
           <property name="title" stdset="0">
            <string>Login</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QWidget" name="widget_11" native="true">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QWidget" name="widget_6" native="true">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 33, 99);</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QWidget" name="widget_8" native="true">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(0, 0, 0);</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_10" stretch="3,3,1">
                     <item>
                      <widget class="QWidget" name="widget_15" native="true"/>
                     </item>
                     <item>
                      <widget class="QWidget" name="widget_7" native="true">
                       <property name="styleSheet">
                        <string notr="true">background-color: rgb(6, 11, 47);</string>
                       </property>
                       <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                         <widget class="QWidget" name="widget_9" native="true">
                          <layout class="QVBoxLayout" name="verticalLayout_9">
                           <item>
                            <widget class="QLabel" name="emailLabel">
                             <property name="font">
                              <font>
                               <family>Script MT Bold</family>
                               <pointsize>12</pointsize>
                               <bold>true</bold>
                              </font>
                             </property>
                             <property name="styleSheet">
                              <string notr="true">color: rgb(25, 255, 21);</string>
                             </property>
                             <property name="text">
                              <string>Email</string>
                             </property>
                            </widget>
                           </item>
                           <item>
                            <widget class="QLabel" name="passwordLabel">
                             <property name="font">
                              <font>
                               <family>Script MT Bold</family>
                               <pointsize>12</pointsize>
                               <bold>true</bold>
                              </font>
                             </property>
                             <property name="styleSheet">
                              <string notr="true">color: rgb(25, 255, 21);</string>
                             </property>
                             <property name="text">
                              <string>Password</string>
                             </property>
                            </widget>
                           </item>
                          </layout>
                         </widget>
                        </item>
                        <item>
                         <widget class="QWidget" name="widget_10" native="true">
                          <layout class="QVBoxLayout" name="verticalLayout_8">
                           <item>
                            <widget class="QLineEdit" name="emailLineEdit">
                             <property name="styleSheet">
                              <string notr="true">background-color: rgb(103, 103, 103);</string>
                             </property>
                            </widget>
                           </item>
                           <item>
                            <widget class="QLineEdit" name="passwordLineEdit">
                             <property name="styleSheet">
                              <string notr="true">background-color: rgb(103, 103, 103);</string>
                             </property>
                             <property name="echoMode">
                              <enum>QLineEdit::Password</enum>
                             </property>
                            </widget>
                           </item>
                          </layout>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </item>
                     <item alignment="Qt::AlignHCenter">
                      <widget class="QWidget" name="widget_12" native="true">
                       <property name="maximumSize">
                        <size>
                         <width>209</width>
                         <height>127</height>
                        </size>
                       </property>
                       <layout class="QVBoxLayout" name="verticalLayout_12">
                        <property name="leftMargin">
                         <number>6</number>
                        </property>
                        <property name="topMargin">
                         <number>6</number>
                        </property>
                        <property name="rightMargin">
                         <number>6</number>
                        </property>
                        <property name="bottomMargin">
                         <number>6</number>
                        </property>
                        <item>
                         <widget class="QPushButton" name="loginButton">
                          <property name="styleSheet">
                           <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
                          </property>
                          <property name="text">
                           <string>Login</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QPushButton" name="signUpButton">
                          <property name="styleSheet">
                           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
                          </property>
                          <property name="text">
                           <string>Sign Up</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="registrationPage">
           <property name="title" stdset="0">
            <string>Register</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 0, 127);</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(85, 85, 127);</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QWidget" name="widget_13" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QWidget" name="widget_4" native="true">
                       <property name="styleSheet">
                        <string notr="true">background-color: rgb(37, 39, 37);</string>
                       </property>
                       <layout class="QVBoxLayout" name="verticalLayout_3">
                        <item>
                         <widget class="QLabel" name="nameLabel">
                          <property name="font">
                           <font>
                            <family>Script MT Bold</family>
                            <pointsize>12</pointsize>
                            <bold>true</bold>
                           </font>
                          </property>
                          <property name="text">
                           <string>Name</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="emailSignUpLabel">
                          <property name="font">
                           <font>
                            <family>Script MT Bold</family>
                            <pointsize>12</pointsize>
                            <bold>true</bold>
                           </font>
                          </property>
                          <property name="text">
                           <string>Email</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="passwordSignUpLabel">
                          <property name="font">
                           <font>
                            <family>Script MT Bold</family>
                            <pointsize>12</pointsize>
                            <bold>true</bold>
                           </font>
                          </property>
                          <property name="text">
                           <string>Password</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </item>
                     <item>
                      <widget class="QWidget" name="widget_5" native="true">
                       <property name="styleSheet">
                        <string notr="true">background-color: rgb(2, 6, 100);</string>
                       </property>
                       <layout class="QVBoxLayout" name="verticalLayout_4">
                        <item>
                         <widget class="QLineEdit" name="nameLineEdit"/>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="emailSignUpLineEdit"/>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="passwordSignUpLineEdit">
                          <property name="echoMode">
                           <enum>QLineEdit::Password</enum>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_14" native="true">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(63, 14, 82);</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_13">
                  <property name="leftMargin">
                   <number>20</number>
                  </property>
                  <property name="rightMargin">
                   <number>20</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="registerButton">
                    <property name="styleSheet">
                     <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
color: rgb(0, 0, 255);
font: 12pt &quot;Segoe Print&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Register</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="chatPage">
           <property name="title" stdset="0">
            <string>Chat</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QWidget" name="widget_16" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 85, 127);</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1">
               <item>
                <widget class="QWidget" name="widget_17" native="true">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(118, 118, 118);</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_15" stretch="3,1">
                  <item>
                   <widget class="QWidget" name="widget_19" native="true">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 85, 0);
background-color: rgb(53, 73, 98);</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_16">
                     <item>
                      <widget class="QTextEdit" name="chatTextEdit">
                       <property name="font">
                        <font>
                         <family>Script MT Bold</family>
                         <pointsize>20</pointsize>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">background-color: rgb(157, 226, 255);
background-color: rgb(42, 42, 42);</string>
                       </property>
                       <property name="readOnly">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_20" native="true">
                    <property name="styleSheet">
                     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 80, 255), stop:0.05 rgba(14, 8, 73, 255), stop:0.36 rgba(28, 17, 145, 255), stop:0.6 rgba(126, 514, 81, 5), stop:0.75 rgba(2348, 119, 11, 255), stop:0.79 rgba(244, 70,4 5, 255), stop:0.86 rgba(955, 186, 0, 255), stop:0.935 rgba(739, 446, 98, 295));</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QLineEdit" name="messageLineEdit">
                       <property name="styleSheet">
                        <string notr="true">background-color: rgb(68, 68, 68);</string>
                       </property>
                       <property name="placeholderText">
                        <string>Text a message</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="sendMessageButton">
                       <property name="styleSheet">
                        <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
color: rgb(0, 156, 0);
font: 20pt &quot;Calisto MT&quot;;</string>
                       </property>
                       <property name="text">
                        <string>Send</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_18" native="true">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(138, 0, 0);
background-color: rgb(0, 0, 127);</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_17">
                  <item>
                   <widget class="QWidget" name="widget_21" native="true">
                    <property name="styleSheet">
                     <string notr="true">image: url(:/new/prefix1/LOGO NYSD.jpg);</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_22" native="true">
                    <property name="styleSheet">
                     <string notr="true">image: url(:/new/prefix1/ictlogo.jpg);</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
